script/test/fixtures/etherpad/docker-compose.yml

```diff
@@ -6,10 +6,10 @@ services:
     ports:
       - "$DB_PORT"
     environment:
-      MYSQL_ROOT_PASSWORD: $ROOT_PASS
-      MYSQL_DATABASE: $DB_NAME
-      MYSQL_PASSWORD: $DB_PASS
-      MYSQL_USER: $DB_USER
```

Comment:
Sstop changing these. Just leave them as they are with the `:`. No need to change ANY of them.